crates/zkvm/zkvm/src/lib.rs

```diff
@@ -204,3 +204,102 @@ impl AggregationInput {
         &self.vk
     }
 }
+
+/// A trait representing a Zero-Knowledge Virtual Machine (ZkVM) interface.
+/// Provides methods for reading inputs, committing outputs, and verifying proofs
+/// within the ZkVM environment.
+pub trait ZkVm {
```

Comment:
Shouldn't this be named something like `ZkVmHost`, `ZkVmInterface`, or `ZkVmIo`?  A name like this makes me think that it's to generically interface with the zkvm runtime from the *outside*.

Reply:
`ZkVmHost` interface is already defined and is meant for outside of zkvm runtime.
Does renaming to `ZkVmEnv` makes sense?

Reply:
That makes sense yeah.